etheno/__main__.py

```diff
@@ -19,52 +19,38 @@
-                             'generic Echidna fuzzing contract)')
-    parser.add_argument('-t', '--truffle', action='store_true', default=False,
-                        help='Run the truffle migrations in the current directory and exit')
+# Constant for converting whole units to wei
+ETHER = 1e18
+
+
+def main(argv = None):
+    parser = argparse.ArgumentParser(description='An Ethereum JSON RPC multiplexer, differential fuzzer, and test framework integration tool.')
+    parser.add_argument('--debug', action='store_true', default=False, help='Enable debugging from within the web server')
+    parser.add_argument('--run-publicly', action='store_true', default=False, help='Allow the web server to accept external connections')
+    parser.add_argument('-p', '--port', type=int, default=GETH_DEFAULT_RPC_PORT, help='Port on which to run the JSON RPC webserver (default=%d)' % GETH_DEFAULT_RPC_PORT)
+    parser.add_argument('-a', '--accounts', type=int, default=10, help='Number of accounts to create in the client (default=10)')
+    parser.add_argument('-b', '--balance', type=float, default=1000.0, help='Default balance (in Ether) to seed to each account (default=100.0)')
+    # TODO: do we really need a gas price specified for ganache? is there a use case here?
+    parser.add_argument('-c', '--gas-price', type=int, default=20000000000, help='Default gas price (default=20000000000)')
+    # TODO: networkID can have a default value it seems like
+    parser.add_argument('-i', '--network-id', type=int, default=None, help='Specify a network ID (default is the network ID of the master client)')
+    '''
+    We might need the Echidna feature later for differential fuzz testing but for reducing confusion we will remove the command-line arguments
+    parser.add_argument('-e', '--echidna', action='store_true', default=False, help='Fuzz the clients using transactions generated by Echidna')
+    parser.add_argument('--fuzz-limit', type=int, default=None, help='The maximum number of transactions for Echidna to generate (default=unlimited)')
+    parser.add_argument('--fuzz-contract', type=str, default=None, help='Path to a Solidity contract to have Echidna use for fuzzing (default is to use a builtin generic Echidna fuzzing contract)')
+    '''
+    parser.add_argument('-t', '--truffle', action='store_true', default=False, help='Run the truffle migrations in the current directory and exit')
     parser.add_argument('--truffle-cmd', type=str, default='truffle', help='Command to run truffle (default=truffle)')
     parser.add_argument('--truffle-args', type=str, default='migrate',
                         help='Arguments to pass to truffle (default=migrate)')
     parser.add_argument('-g', '--ganache', action='store_true', default=False,
                         help='Run Ganache as a master JSON RPC client (cannot be used in conjunction with --master)')
+    # TODO: This cmd-line argument is error-prone and should probably be removed. Commenting it out for now
```

Comment:
how is it error prone?

Reply:
the `ganache-cmd` by default is `/usr/bin/env ganache` and then you have `ganache-port` and then `ganache-args`. However, `ganache-cmd` could encompass all three (`/usr/bin/env ganache -p 1234 --gasLimit blah blah`) and if one of the other ones are also specified it might cause ganache to not start up. By removing it, everything is broken apart into independent pieces. The ability for `ganache-cmd` to overwrite everything else seems unnecessary...I guess error-prone is not the best language for this.